packages/plugin-analytics/config/robo.mjs

```diff
@@ -0,0 +1,12 @@
+// @ts-check
+
+/**
+ * @type {import('robo.js').Config}
+ **/
+export default {
+	clientOptions: {
+		intents: ['Guilds', 'GuildMessages']
```

Comment:
Please remove these intents from the plugin. We'll only log events projects have access to.